### ISSUE TITLE
Watching specific namespaces and multiple operators running independently within a same cluster

### ISSUE THREAD

**Issue by navoyan** [zilliztech/milvus-operator#190]
Hi!
We have isolated (as much as possible) environments in our cluster that are deployed to different namespaces. This is possible because all the operators we currently use are configured to watch only one namespace. That is, they do not steal the work of other operators in other namespaces, nor do they watch resource changes in other namespaces.

It seems impossible to achieve the same with the milvus operator. It makes cluster-scoped Kubernetes calls, which means that limiting it's permissions via namespaced RBAC won't do the trick.

Am I missing something? If not, is there any support for this feature planned?

**Comment by haorenfsa** [comment]
You're right, It's not supported now. And no plan for this, since it seems not a very common requirement for community users. 

You can make a patch to implement it though. There won't be much work in my consideration. You just need to add a  flag, and make controller only watch the specified namespace.
